Compiler/include/Luau/Compiler.h

```diff
@@ -36,6 +36,10 @@ struct CompileOptions
     // global builtin to construct vectors; disabled by default
     const char* vectorLib = nullptr;
     const char* vectorCtor = nullptr;
+
+    // array of globals that are mutable; disables the import optimization for fields accessed through them
+    // use NULL to end the array
+    const char** mutableGlobalNames = nullptr;
```

Comment:
`mutableGlobals`?

Reply:
> `mutableGlobals`?

That was actually my first name for it. I'll go ahead and change it.